README.md

```diff
@@ -0,0 +1,48 @@
+This is a tool to download and create labeled audio datasets from YouTube videos.
+
+You need to execute two steps:
+
+### First step: download audio files and subtitles and convert audio format
+
+ - Add the YouTube video URLs to the `download.txt` file. The selected videos must have subtitles for the desired language.
+
+ - Run the script `./download.sh <subtitle lang>`, where `<subtitle lang>` is the two-letter language code, e.g. `pt`, `es`, `en`, `ru`, etc. 
+
```

Comment:
It might be nice to include a warning that the `downloads` directory will be recreated each time the script is run. E.g. **WARNING**: running this script will cause the `downloads` directory to be recreated each time, erasing any existing data.

Reply:
I added the yes/no bash confirmation.

I think this is enough about the recreated directory warning.

What do you think?